hack/.golint_failures

```diff
@@ -9,6 +9,7 @@ cmd/kubeadm/app/apis/kubeadm/v1alpha1
 cmd/kubelet/app
 cmd/kubelet/app/options
 cmd/kubemark
+examples/guestbook-go
```

Comment:
this package no longer exists. Can you please remove this and squash into a single commit?

Reply:
Done @cblecker